Description: add encoding to POD, Pod::Simple is more strict now
Origin: vendor
Bug: https://rt.cpan.org/Ticket/Display.html?id=85966
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=85966
Bug-Debian: http://bugs.debian.org/711394
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2013-06-18

--- a/lib/User/Simple.pm
+++ b/lib/User/Simple.pm
@@ -3,6 +3,8 @@
 
 package User::Simple;
 
+=encoding UTF-8
+
 =head1 NAME
 
 User::Simple - Simple user sessions management
--- a/lib/User/Simple/Admin.pm
+++ b/lib/User/Simple/Admin.pm
@@ -3,6 +3,8 @@
 
 package User::Simple::Admin;
 
+=encoding UTF-8
+
 =head1 NAME
 
 User::Simple::Admin - User::Simple user administration
